### PR TITLE
Exercise 6

### PR DESCRIPTION
**Pull request by MichaelCollingwood**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: architecture/context.png
Comment (Jonesey13) [inline architecture/context.png:1]: Looks good (could have maybe mentioned their browser as well)

File: architecture/container.png
Comment (Jonesey13) [inline architecture/container.png:1]: Still probably want to keep in the client in this diagram

File: architecture/component.png
Comment (Jonesey13) [inline architecture/component.png:1]: Again looks good but we're missing the requests to the application in this picture

**Review by Jonesey13 — APPROVED** [review]
All the diagrams here look great. Only major observation is that the user/requests are missing from the component/container diagrams.

File: architecture/context.png
Comment (MichaelCollingwood) [reply architecture/context.png:1]: I was being a bit light, I think you're right I could have done into more depth

**Pull request merged**